post-meta-inspector.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
I'd rather see the class moved to its own `class-post-meta-inspector.php` file (a WPCS and PSR-12 rule), and required in this file, rather than having the PHPCS ignore here.